includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -336,43 +336,46 @@ public function data_table_header() {
 			$selected_course = (int) $_GET['filter_by_course_id']; // phpcs:ignore WordPress.Security.NonceVerification
 		}
 		?>
-		<div class="tablenav top">
-			<div class="alignleft actions bulkactions">
-				<div id="sensei-bulk-learner-actions-modal" style="display:none;">
-					<?php $this->render_bulk_actions_form( $courses ); ?>
-				</div>
-				<?php
-				echo wp_kses(
-					$this->render_bulk_action_select_box(),
-					array(
-						'option' => array(
-							'value' => array(),
-						),
-						'select' => array(
-							'id'   => array(),
-							'name' => array(),
-						),
-					)
-				);
-				?>
-				<button type="submit" id="sensei-bulk-learner-actions-modal-toggle" class="button button-primary action"><?php echo esc_html__( 'Select Courses', 'sensei-lms' ); ?></button>
+		<div class="sensei-bulkactions-wrapper">
+			<div class="alignleft actions bulkactions sensei-bulkactions-container">
+			<div id="sensei-bulk-learner-actions-modal" style="display:none;">
+				<?php $this->render_bulk_actions_form( $courses ); ?>
 			</div>
-			<div class="alignleft actions">
-				<form action="" method="get">
+			<div class="sensei-wp-table-filters">
```

Comment:
This class name is pretty generic and doesn't tell me much about the element that is being targeted. Something like `sensei-students-bulk-actions` could be a better identifier, and would allow for bulk actions to be added to some other page besides Students in the future.

Reply:
I did two renamings: 
First [one](https://github.com/Automattic/sensei/pull/4997/commits/affbfbd4646a9c414ea0c0f358174d05dbb3ab67)

But after that I went with this approach so this is the final version: [This](https://github.com/Automattic/sensei/pull/4997/commits/64329a9a4738986852b06fd1da4bb645435722b8)
Followed by these two fixes: [1](https://github.com/Automattic/sensei/pull/4997/commits/c036ac86aa098db5df47fc95c494cac7f51129a0) [2](https://github.com/Automattic/sensei/pull/4997/commits/ec7f95a646aebac6922c4cece05efc747ac11a9f)